src/api/pastel-rpc/transaction.ts

```diff
@@ -0,0 +1,311 @@
+import { remote } from 'electron'
```

Comment:
The remote module is deprecated. See more detail here: https://www.electronjs.org/docs/api/remote 

Can we use ipcRenderer and ipcMain?

Reply:
I can not find the proper method. could you please advise more?

Reply:
I have seen you have removed the `remote`. So it can be resolve now.